charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml

```diff
@@ -0,0 +1,13 @@
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+systemUpgradeController:
+  image:
+    repository: rancher/system-upgrade-controller
+    tag: v0.9.1
```

Comment:
@Oats87 something seems to have gone wrong with the process, I expected this to be v0.10.0?